packages/@glimmer/runtime/test/partial-test.ts

```diff
@@ -23,7 +23,7 @@ function compile(template: string) {
 
 function commonSetup() {
   env = new TestEnvironment();
-  root = document.createElement('div');
+  root = document.getElementById('qunit-fixture')!;
```

Comment:
Unclear if this can result in attribute changes, if not then this is totally good (no need to add a nested child).

Reply:
Yea there shouldn't be any attribute setting.